package.json

```diff
@@ -76,64 +76,64 @@
     },
     "resolutions": {
         "@types/react-native": "0.70.3",
-        "typescript": "4.9.3",
+        "typescript": "4.9.5",
         "react-native": "0.70.5",
-        "prettier": "2.8.0",
+        "prettier": "2.8.4",
         "type-fest": "2.12.2",
         "bcrypto": "5.4.0",
         "react": "18.2.0",
-        "electron": "20.3.5",
+        "electron": "20.3.12",
```

Comment:
What about updating electron to 22?  https://github.com/trezor/trezor-suite/issues/7172

Reply:
https://github.com/trezor/trezor-suite/pull/7626